examples/scripts/ct_astra_weighted_tv_admm.py

```diff
@@ -139,7 +139,7 @@ def postprocess(x):
 
 weights = counts / Io  # scale by Io to balance the data vs regularization term
 W = linop.Diagonal(snp.sqrt(weights))
```

Comment:
Should't the `snp.sqrt` be dropped here?

Reply:
Good catch! Addressed in 911e846.